miden-lib/src/tests/test_account.rs

```diff
@@ -326,13 +326,100 @@ fn test_set_item() {
     let _process = run_tx(&transaction).unwrap();
 }
 
-// TODO: reenable once storage map support is implemented
-#[ignore]
+// Test different account storage types
+#[test]
+fn test_get_storage_data_type() {
+    for storage_item in [storage_item_0(), storage_item_1(), storage_item_2()] {
+        let (tx_inputs, tx_args) =
+            mock_inputs(MockAccountType::StandardExisting, AssetPreservationStatus::Preserved);
+
+        let code = format!(
+            "
+        use.miden::kernels::tx::account
+        use.miden::kernels::tx::prologue
+
+        begin
+            # prepare the transaction
+            exec.prologue::prepare_transaction
+
+            # push the account storage item index
+            push.{item_index}
+
+            # get the data type of the respective storage slot
+            exec.account::get_storage_slot_type_info
+
+        end
+        ",
+            item_index = storage_item.index,
+        );
+
+        let transaction =
+            prepare_transaction(tx_inputs.clone(), tx_args.clone(), code.as_str(), None);
+        let _process = run_tx(&transaction).unwrap();
+        let transaction = prepare_transaction(tx_inputs, tx_args, &code, None);
+        let process = run_tx(&transaction).unwrap();
```

Comment:
Why is this run twice? Can we remove the top call and have just the bottom one?

Reply:
good point, didn't realize - leftover from debugging I guess